### PR TITLE
Update picoquic to fix linux build issue

### PR DESCRIPTION
**Pull request by TimEvens**
This also updates picoquic to fix a linux build issue.

### CODE REVIEW

File: src/transport.cpp [hunk 1]
```diff
@@ -920,9 +920,10 @@ namespace quicr {
                       {
                         msg.group_id,
                         msg.object_id,
+                        msg.subscribe_id,
                         msg.payload.size(),
                         msg.priority,
-                        std::nullopt,
+                        msg.priority,
```
Comment (GhostofCookie) [inline src/transport.cpp:926]: Priority twice?

File: src/transport_picoquic.cpp [hunk 2]
```diff
@@ -478,8 +478,8 @@ PicoQuicTransport::Start()
      *    also triggers PMTUD to run. This value will be the initial value.
      */
     picoquic_init_transport_parameters(&local_tp_options_, 1);
-    local_tp_options_.max_datagram_frame_size = 1280;
-    //  local_tp_options.max_packet_size = 1450;
+    // local_tp_options_.max_datagram_frame_size = 1280;
+    //   local_tp_options.max_packet_size = 1450;
```
Comment (GhostofCookie) [inline src/transport_picoquic.cpp:482]: Dead code?

**Review by GhostofCookie — APPROVED** [review]
Just a few comments, but overall looks good.

File: src/transport_picoquic.cpp (hunk 2 above)
Comment (TimEvens) [reply src/transport_picoquic.cpp:482]: It was more of a note, but I can remove it.

Comment (suhasHere) [reply src/transport_picoquic.cpp:482]: +1 to rmeoving this

File: src/transport.cpp (hunk 1 above)
Comment (TimEvens) [reply src/transport.cpp:926]: good catch. It was supposed to still be nullopt.  I've updated that.

**Review by RichLogan — APPROVED** [review]
LGTM

**Review by GhostofCookie — APPROVED** [review]

**Pull request merged**